docs/annotations.md

```diff
@@ -0,0 +1,55 @@
+# Annotations
+Annotations are a way for students (and possibly teachers or CLUE itself in the future) to add additional information to their documents. Annotations are tile independent, so they can be added to anything within CLUE that has been properly set up, and can even span between multiple tiles.
+
+## Types of Annotations
+Currently there is just a single type of annotation, the student proportional arrow, or sparrow. In the future, there could be many different types of annotations, such as comments on a single object, circles for emphasis, Xs to indicate mistakes, etc.
+
+### Sparrows
+Student proportional arrows, more commonly called sparrows, connect two objects with an arrow and a text box. One of the objects is the source, which has the tail end of the arrow, and the other is the target, which gets the arrowhead. The text box can be moved around and defines the curve of the arrow.
+
+## Setting up Tiles to use Annotations
+Each tile must be set up to define which objects within it can be annotated.
+
+### CLUE Objects
+Annotations can be connected to anything set up as a CLUE Object. These can be anything within a tile, and tiles themselves are in charge of interpretting what they are. A CLUE Object is defined by three pieces of information:
+- tileId: This is the actual id of the tile containing the object.
+- objectId: This is a unique way to identify the object from the tile's perspective. This will often be the object's actual id, but it doesn't need to be. For example, cells in table tiles have no ids, so the objectId of a cell is a string that contain's the cell's case and attribute ids.
```

Comment:
```suggestion
- objectId: This is a unique way to identify the object from the tile's perspective. This will often be the object's actual id, but it doesn't need to be. For example, cells in table tiles have no ids, so the objectId of a cell is a string that contains the cell's case and attribute ids.
```